ig/input/fsh/logical-orm-O01.fsh

```diff
@@ -0,0 +1,59 @@
+// defines the ORM-O01 (loosely) input that is expected for newborn screening
+
+//TODO: lines 11-12; Do we want to cross refernce the MSH and PID Segments fron the ADT message, or should they be custom to the ORM?
```

Comment:
I think for now they can be the same.  Consider moving the `MSH` and `PID` segments to a common/shared file instead of the `logical-adt-01.fsh` file.